src/M161.md

```diff
@@ -0,0 +1,38 @@
+# M161
+
+The M161 is a simple multi-cart mapper by Mani (Nintendo's official Chinese distributor). 
+This mapper only allows for 8 separate 32 KiB banks, with a limit of 1 bankswitch, preventing any bankswitches afterward.
+This gives 1 bank for the main menu, and a maximum of 7 different 32 KiB games. No SRAM can be applied to this mapper, so only purely MBC-less titles can be used.
+
+This mapper is only known to be used in a single cartridge: Mani's 4 in 1 cartridge with Tetris, Tennis, Alleyway, and Yakuman.
+All later Mani 4 in 1 cartridges use the MMM01 mapper, so the M161 mapper could be considered the predecessor to the MMM01.
+
+The mapper is really just a simple off-the-shelf 74HC161A used a flip-flop. The 74HC161A contains 4 input pins and 4 output pins, and another pin for latching the input pins to the output pins.
+The first 3 of the input pins are connected to the lower 3-bits of incoming write data, with the 4th pin kept high. The first 3 of the output pins are connected to the ROM addressing pins.
+The 4th output pin (which corresponds to the always high input pin) is connected to the latching pin. Bit-15 of the write address is also connected (OR'd) with the latching pin.
+The latching pin latches the input pins to the output pins on a high to low, so when a ROM write comes through (bit-15 = 0), the input pins will be latched to the output pins.
+Since 4th output pin will be high, this will render it no longer possible for a high to low latch signal to occur, thus preventing further bankswitches.
+
+
+## Memory
+
+### 0000-7FFF - ROM Bank $00-$07 (Read Only)
+
+This area contains a 32 KiB bank of ROM. On startup, this will contain the first 32 KiB of ROM (bank 0).
+
+## Registers
+
+### 0000-7FFF - ROM Bank Number (Write Only)
+
+This 3-bit register (range $00-$07) selects the ROM bank number for the $0000-$7FFF region. Like other mappers, the high bits are discarded.
+
+Only 1 bankswitch is allowed per session, once any write to this register occurs (even to bank 0), all future writes will do nothing until a hard reset.
+
+## Operation Notes
+
+Because the full 32 KiB range is switched over, caution should be taken, as the entire ROM range will change.
+It may be best to have the bankswitch code placed right before $0100, as to slide down to a game's init sequence.
```

Comment:
```suggestion
It may be best to have the bankswitch code placed right before $0100, so as to fall through to [the game's init sequence](https://gbdev.io/pandocs/The_Cartridge_Header.html#0100-0103---entry-point).
```